scaladoc/test-documentations/basic/index.md

```diff
@@ -1,8 +0,0 @@
----
-title: Basic test
----
-# Header
-
-[Link to docs](docs/index.html)
-
-And a text!
```

Comment:
The file is removed because it's not used anymore - there was kind of mess with indexes: we were able to define docs index, api index and landing page. With my changes, the landing page is always either api index or docs index depending on the `-Yapi-subdirectory` setting. The reason is that I think the root index of the documentation should be a doc page already. Custom landing pages are not necessarily a responsibility of doctool.